crates/breez-sdk/core/src/models/mod.rs

```diff
@@ -346,6 +346,16 @@ pub struct Config {
 
     /// Url to use for the real-time sync server. Defaults to the Breez real-time sync server.
     pub real_time_sync_server_url: Option<String>,
+
+    /// The initial user settings to be applied.
+    ///
+    /// This configuration is only relevant when the SDK is initialized for the first time for any given seed.
+    /// To adjust the settings after that, use [`BreezSdk::update_user_settings`](crate::sdk::BreezSdk::update_user_settings).
+    /// To get the current settings, use [`BreezSdk::get_user_settings`](crate::sdk::BreezSdk::get_user_settings).
```

Comment:
Is this over all devices/instances or the current device? What I'm asking really is if it's for all devices how does the partner/implementer know they should call update_user_settings instead?

Reply:
If the partner knows the seed is new, they can take advantage of this configuration. If not, and they need some user setting to be applied, they will need to use `update_user_settings`.